### DISCUSSION TITLE
How to customize the background of panel

### DISCUSSION THREAD

**Discussion by piyushdongre-dot-com** [filamentphp/filament#8818] [Help] [answered]
Hi,
I want to change background of the Panel, with some svg. Also if the background can be customized at the `resource` or `page` level ?.

I have looked into publishing filament theme. But don't know how to change background.

Thanks

**Comment by VincvanAs** [comment]
You could create a view and render a hook by adding:

`->renderHook('panels::body.start', fn (): View => view('filament.hooks.background'));`

To your PanelProvider.

**Reply by leandrocfe** [reply, marked as answer]
[creating-a-custom-theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) and editing the css?

**Reply by piyushdongre-dot-com** [reply]
Yes, theme works for a global background by using
.fi-body class

**Reply by leandrocfe** [reply]
Sorry, I didn't understand the issue

**Comment by rthomas98** [comment]
There is the base.blade.php file that you can edit.
